packages/eslint-plugin/src/rules/naming-convention-utils/enums.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/prefer-literal-enum-member */
```

Comment:
we should probably be explicit with the enums in this file

Reply:
Just confirming - do you mean you're not in favor of the bitwise `1 << 0`s, etc.?

Reply:
@bradzacher was there any action you wanted me to take here?